pandas/core/frame.py

```diff
@@ -2129,7 +2129,7 @@ def to_stata(
             characters or fewer while versions 117 and later allow strings
             with lengths up to 2,000,000 characters. Versions 118 and 119
             support Unicode characters, and version 119 supports more than
-            32,767 variables.
```

Comment:
I wouldn't say "for now". Something along the lines of

"Version 119 should usually only be used when the number of variables exceeds the capacity of the dta version 118. Exporting smaller datasets to format 119 files may have unintended consequences, and, as of November 2020, Stata SE cannot read version 119 files. "

Reply:
Thanks,

I copied your suggestion verbatim.